pyanaconda/modules/common/structures/group.py

```diff
@@ -50,7 +50,7 @@ def name(self, name: Str):
         self._name = name
 
     @property
-    def gid(self) -> Int:
+    def gid(self) -> Int64:
```

Comment:
That's still not correct -- u/gids can never be negative, nor so large. Can you please use a proper `UInt32` instead?